client/components/phone-input/index.jsx

```diff
@@ -243,6 +243,7 @@ class PhoneInput extends React.PureComponent {
 					name={ this.props.name }
 					ref={ this.setNumberInputRef }
 					type="tel"
+					className={ this.props.isError && 'is-error' }
```

Comment:
For future reference - I understand that it's tempting to write it like this, but it does not work as you'd expect it to.
Using `classnames` is preferred, because it handles false-y values properly (by not appending anything in those cases), whereas when `this.props.isError` is `false`, this expression is evaluated to the string `'false'`. You can see it when you inspect the element when there's no error:
<img width="539" alt="screen shot 2017-11-16 at 13 33 38" src="https://user-images.githubusercontent.com/3392497/32891491-03e37558-cad3-11e7-8a31-bde9605df16c.png">

Reply:
Thanks @klimeryk. Now that you mention it, I should have caught this from previous experience. I'll fix it in another PR. 👍